tests/test_app_metadata.py

```diff
@@ -191,6 +191,8 @@ def test_app_metadata(
     expected_metadata: Dict,
     application_executable: str,
 ) -> None:
+    if runtime == "dotnet":
+        pytest.xfail("Dotnet-trace doesn't work with alpine: https://github.com/Granulate/gprofiler/issues/795")
```

Comment:
So by switching the base layer of the container to alpine, we break .NET completely for the container build? (not saying it's a problem, as .NET hasn't been ours/users' focus lately, just trying to understand)

Reply:
Unfortunately yes :(